app/static/css/style.css

```diff
@@ -197,7 +197,7 @@ textarea {
 }
 
 #footer {
-    position: absolute;
+    position: fixed;
```

Comment:
Remove this line. This fix has been already provided in other PR.

Reply:
I have said him to close that PR, as he broke the rules.